src/webui/www/private/shareratio.html

```diff
@@ -0,0 +1,167 @@
+<!DOCTYPE html>
+<html lang="${LANG}">
+<head>
+    <meta http-equiv="Content-Type" content="text/html; charset=UTF-8" />
```

Comment:
PR #8621 is merged now, can you please apply the changes to here?

Reply:
Updated to reflect the changes.